### ISSUE TITLE
Условия сотрудничества с платными конференциями

### ISSUE THREAD

**Issue by flpvsk** [MoscowJS/moscowjs.ru#13]
Составить документ, в котором описать что мы можем предложить платным конференциям и что они могут предложить нам.
#### Заметки со встречи организаторов

Мы:
- Поиск докладчиков
- Подготовка докладчиков: прогоны, советы по слайдам
- Реклама в рассылке, fb, twitter

Конференция:
- Бесплатные билеты
- Реклама митапа
- Деньги индивидуально (тем кто прогоняет например)

**Comment by chicoxyzzy** [comment]
так, я наконец-то нашел время заняться этим. @MoscowJS/owners может кто-нибудь хочет что-то добавить?
